docker-wrappers/AllPairs/sample-in-net.txt

```diff
@@ -0,0 +1,21 @@
+#Node1	Node2
```

Comment:
Are these the same files in the test subdirectories? Do we need the extra copies here as well or can they be deleted?

Reply:
I deleted them, and will add a note in the README where to find the format of the files